internal/scan-manager/scanners/veracode/src/main/java/org/wso2/security/tools/scanmanager/scanners/veracode/handler/ScanTask.java

```
@@ -347,8 +352,20 @@ private void copyRequiredScanArtifact(String filePath) throws IOException, SAXEx
             ParserConfigurationException {
         File dir = new File(filePath);
         File[] files = dir.listFiles();
-        File patternXmlFile = new File(VeracodeScannerConfiguration.getInstance().getConfigProperty(
-                VeracodeScannerConstants.JAR_FILTER_PATTERN_FILE_PATH));
+        File patternXmlFile = new File(JAR_FILTER_FILE);
+
+        try (InputStream input = VeracodeScannerConfiguration.class.getClassLoader()
+                .getResourceAsStream(JAR_FILTER_FILE);
+             OutputStream out = new FileOutputStream(patternXmlFile)) {
+            int read;
+            byte[] bytes = new byte[1024];
+
+            while ((read = input.read(bytes)) != -1) {
+                out.write(bytes, 0, read);
+            }
+        } catch (IOException e) {
+            throw new IOException(e);
```

Better to throw the exception with a meanigful error message also.